flink/src/main/java/org/apache/iceberg/flink/data/FlinkOrcReaders.java

```diff
@@ -49,19 +49,19 @@
 import org.apache.orc.storage.ql.exec.vector.TimestampColumnVector;
 import org.apache.orc.storage.serde2.io.HiveDecimalWritable;
 
-class FlinkOrcReaders {
+public class FlinkOrcReaders {
   private FlinkOrcReaders() {
   }
 
-  static OrcValueReader<StringData> strings() {
+  public static OrcValueReader<StringData> strings() {
```

Comment:
curious why only making these 3 public. what about other package scope methods like `times`, `timestamps`, `array`?

Reply:
it is my mistake, I rolled back it